bigquery/schema_builder.py

```diff
@@ -126,7 +126,7 @@ def bigquery_type(o, timestamp_parser=default_timestamp_parser):
     """
 
     t = type(o)
-    if t == int:
+    if isinstance(t, six.integer_types):
```

Comment:
Since `t` is a `type`, I don't think this is quite right. I believe this just needs to be `if t in six.integer_types` since `six.integer_types` is a tuple containing all integer types.